media-libs/giflib/files/giflib-5.1.9-fix-missing-quantize-api-symbols.patch

```diff
@@ -0,0 +1,32 @@
+From ff8d9a59e79b79657e64430730c35835a84db619 Mon Sep 17 00:00:00 2001
+From: anthraxx <levente@leventepolyak.net>
+Date: Tue, 2 Apr 2019 11:46:18 +0200
+Subject: [PATCH] fix missing quantize API symbols
+
+GifQuantizeBuffer is required by many libs and applications
+like mplayer, libgdiplus (mono) and others.
+---
+ Makefile | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
```

Comment:
This patch might be a sufficient workaround for `5.1.9`, but `>=5.2.0` contains this upstream commit:
https://sourceforge.net/p/giflib/code/ci/befe188771035de7acbc2ad46bea22ea73ab09cd/

For a workaround for `>=5.2.0`, the move of `GifQuantizeBuffer` declaration from `gif_lib.h` and `getarg.h` would have to be reverted.

Reply:
Okay, added to 5.2.1-r1.